ts-client/src/dlmm/index.ts

```diff
@@ -4303,6 +4279,39 @@ export class DLMM {
     }).add(syncWithMarketPriceTx);
   }
 
+  public async getMaxPrice(
+    minPrice: string,
```

Comment:
why we need input minPrice here? the function getMaxPrice doesn't need input parameter as minPrice

Reply:
that's true, we dont need this